benchmarks/imagenet/resnet50/main.py

```diff
@@ -230,7 +230,7 @@ def pretrain(
         logger=TensorBoardLogger(save_dir=str(log_dir), name="pretrain"),
         precision=precision,
         strategy="ddp_find_unused_parameters_true",
-        sync_batchnorm=True,
+        sync_batchnorm=accelerator != "cpu",  # Sync batchnorm is not supported on CPU.
```

Comment:
Added this to make benchmarks runnable on CPU, makes testing distributed training easier.